src/core/credentials/transport/tls/private_key_offload_util.h

```diff
@@ -81,70 +73,15 @@ struct TlsPrivateKeyOffloadContext {
 
 // Callback function to be invoked when the user's async sign operation is
 // complete.
-
-static void TlsOffloadSignDoneCallback(
-    TlsPrivateKeyOffloadContext* ctx, absl::StatusOr<std::string> signed_data) {
-  if (signed_data.ok()) {
-    ctx->signed_bytes = std::move(signed_data);
-
-    // Notify the TSI layer to re-enter the handshake.
-    // This call is thread-safe as per TSI requirements for the callback.
-    if (ctx->notify_cb) {
-      std::string bytes_to_send = *ctx->signed_bytes;
-      const unsigned char* bytes_to_send_ptr =
-          reinterpret_cast<const unsigned char*>(bytes_to_send.c_str());
-      ctx->notify_cb(TSI_OK, ctx->notify_user_data, bytes_to_send_ptr,
-                     bytes_to_send.length(), *ctx->handshaker_result);
-    }
-  } else {
-    ctx->signed_bytes = signed_data.status();
-    // Notify the TSI layer to re-enter the handshake.
-    // This call is thread-safe as per TSI requirements for the callback.
-    if (ctx->notify_cb) {
-      ctx->notify_cb(TSI_INTERNAL_ERROR, ctx->notify_user_data, nullptr, 0,
-                     *ctx->handshaker_result);
-    }
-  }
-}
+static void TlsOffloadSignDoneCallback(TlsPrivateKeyOffloadContext* ctx,
```

Comment:
FYI it looks like you moved the impls to a `.cc` file, but the `.cc` file hasn't been added to the git commit yet